src/libs/kata-types/src/config/agent.rs

```diff
@@ -75,6 +75,14 @@ pub struct Agent {
     pub container_pipe_size: u32,
 }
 
+fn default_server_port() -> u32 {
+    1024
+}
+
+fn default_log_port() -> u32 {
+    1025
```

Comment:
Why this is the default value?

Reply:
That is how it was written, even code included in the go version. By convention, the kata agent server is connected in port 1024.  And we add a log port at 1025.

Reply:
Can we define these values once then? For example `default_server_port()` and the agent should both uses a common `const` (aka move [`VSOCK_PORT`](https://github.com/kata-containers/kata-containers/blob/main/src/agent/src/config.rs#L31) to somewhere under `src/libs/`).